CRM/Core/Form/Search.php

```diff
@@ -304,7 +304,12 @@ protected function getEntityDefaults($entity) {
       if (empty($_POST[$fieldName])) {
         $value = CRM_Utils_Request::retrieveValue($fieldName, $this->getValidationTypeForField($entity, $fieldName), NULL, NULL, 'GET');
         if ($value !== NULL) {
-          $defaults[$fieldName] = $value;
+          if ($fieldSpec['html']['type'] === 'Select') {
```

Comment:
@eileenmcnaughton this works for me, however i do have a question on this, should we be diferenciating multi select v non multiselect e.g. on the form currently event type is not multiselect but participant status is. Woud the thinking be that all selects go multiselect?

Reply:
Yeah - multiselect is kinda Select now on the search form isn't it?

Reply:
I think so or at the very least there is nothing in the metadata that would tell you if it was multi or single select (i think all being multiselect on search forms at least makes very good sense tbh but if we were using the same metadata to say build admin form it might be a different thing right?)

Reply:
Yeah - I think it's one to keep pondering but I would only open it up as we come to it

Reply:
Yeh agreed,